File: datalib/src/main/java/nl/acidcats/tumblrlikes/core/usecases/photos/UpdatePhotoCacheUseCase.java
package nl.acidcats.tumblrlikes.core.usecases.photos;

import rx.Observable;

/**
 * Created on 01/10/2018.
 */
public interface UpdatePhotoCacheUseCase {
    Observable<Boolean> removeCachedHiddenPhotos();

    Observable<Integer> checkCachedPhotos();
}

File: datalib/src/main/kotlin/nl/acidcats/tumblrlikes/core/usecases/photos/UpdatePhotoCacheUseCase.kt
package nl.acidcats.tumblrlikes.core.usecases.photos

import rx.Observable

/**
 * Created on 01/10/2018.
 */
interface UpdatePhotoCacheUseCase {
    fun removeCachedHiddenPhotos(): Observable<Boolean>

    fun checkCachedPhotos(): Observable<Int>
}
